src/coreclr/jit/importer.cpp

```diff
@@ -4298,6 +4298,25 @@ GenTree* Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolvedT
             break;
         }
 
+        case CORINFO_FIELD_STATIC_DATASEGMENT:
+        {
+#ifdef FEATURE_READYTORUN
+            assert(opts.IsReadyToRun());
+            assert((pFieldInfo->fieldFlags & CORINFO_FLG_FIELD_INITCLASS) == 0);
+            assert(pFieldInfo->fieldLookup.accessType == InfoAccessType::IAT_VALUE);
+            assert(fieldKind == FieldSeq::FieldKind::SimpleStatic);
+
+            op1         = gtNewIconHandleNode((size_t)pFieldInfo->fieldLookup.addr, GTF_ICON_STATIC_HDL);
+            op1->gtType = TYP_BYREF;
```

Comment:
Can this just an untracked pointer?

Reply:
The Jit will not fold handles when reporting relocs modulo bugs (see e.g. `ImmedValCanBeFolded`).

Reply:
Yeah looks like it doesn't fold, right